backends/tc/runtime/clone-iproute2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RUNTIME_DIR=$1
+
+git clone https://github.com/p4tc-dev/iproute2-p4tc-pub.git $RUNTIME_DIR/iproute2-p4tc-pub
```

Comment:
You could use CMake's fetchcontent utilities for this. See also how the eBPF back end does it. https://github.com/p4lang/p4c/blob/main/backends/ebpf/CMakeLists.txt#L20

Reply:
Ohh, okk
I'm not very familiar with cmake
Will copy what you did there

Reply:
Added it here: https://github.com/vbnogueira/p4c/blob/automated_p4tc_tests/backends/tc/CMakeLists.txt#L17